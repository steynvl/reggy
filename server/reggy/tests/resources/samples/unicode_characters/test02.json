{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Unicode characters",
        "markedStrings": [
          "das"
        ],
        "markerInfo": {
          "lowercaseLetters": false,
          "uppercaseLetters": true,
          "titleCaseLetters": false,
          "casedLetters": false,
          "modifierLetters": false,
          "otherLetters": false,
          "nonSpacingMarks": false,
          "spacingCombiningMarks": false,
          "enclosingMarks": false,
          "spaceSeparators": false,
          "lineSeparators": true,
          "paragraphSeparators": true,
          "mathSymbols": false,
          "currencySymbols": false,
          "modifierSymbols": false,
          "otherSymbols": false,
          "decimalDigitNumbers": false,
          "letterNumbers": false,
          "otherNumbers": false,
          "dashPunctuation": true,
          "openPunctuation": true,
          "closePunctuation": true,
          "initialPunctuation": false,
          "finalPunctuation": false,
          "connectorPunctuation": false,
          "otherPunctuation": false,
          "controlCharacters": false,
          "formatCharacters": false,
          "privateUseCharacters": false,
          "surrogateCharacters": false,
          "unassignedCharacters": false,
          "matchAllExceptSelectedOnes": false,
          "individualCharacters": ""
        },
        "repeatInfo": {
          "repeat": "6"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
