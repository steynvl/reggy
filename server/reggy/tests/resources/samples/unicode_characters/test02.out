Java
Pattern regex = Pattern.compile("[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}");
[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}
Perl
my $regex = /[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}/;
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
POSIX
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
Python
regex = re.compile(r'[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}')
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
JavaScript
const regex = /[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}/;
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
PHP
$regex = '/[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}/';
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
Golang
regex, _ := regexp.Compile("[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}")
[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}
Rust
let re = Regex::new(r"[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}").unwrap();
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
C#
var re = new Regex(@"[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}");
[\p{Lu}\p{Zl}\p{Zp}\p{Pd}\p{Ps}\p{Pe}]{6}
Scala
val re = "[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}".r
[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}
Kotlin
val regex = Regex("[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}")
[\\p{Lu}\\p{Zl}\\p{Zp}\\p{Pd}\\p{Ps}\\p{Pe}]{6}